src/ngsw-worker.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright © Live-Poll 2020-2021. All rights reserved
+ */
+
+const CACHE_NAME = 'static-cache';
+const URLS_TO_CACHE = [
+  '/',
+  '/index.html',
+  '/styles.css',
+  '/assets/themes/light.css',
+  '/assets/themes/dark.css',
+  '/assets/icons/apple-icon-180.png',
+  '/assets/icons/manifest-icon-192.png',
+  '/assets/icons/manifest-icon-512.png',
+  '/assets/images/drag_indicator.svg',
+  '/assets/images/empty.svg',
+  '/assets/images/logo.svg',
+  '/assets/images/light/wave-bot.svg',
+  '/assets/images/light/wave-mid.svg',
+  '/assets/images/light/wave-top.svg',
+  '/assets/images/dark/wave-bot.svg',
+  '/assets/images/dark/wave-mid.svg',
+  '/assets/images/dark/wave-top.svg'
+];
+
+self.addEventListener('push', function (event) {
+  if (self.Notification && self.Notification.permission === 'granted') {
+    if (event.data !== undefined && event.data !== null) {
+      const data = event.data.json();
+      // Send push notification to users system
+      registration.showNotification(data.title, {
+        body: data.message,
+        tag: 'push-notification',
+        icon: 'assets/icons/manifest-icon-512.png'
+      });
+    } else {
+      console.log('Push notification data parsing failed.');
+    }
+  }
+});
+
+self.addEventListener('notificationclick', function (event) {
+  const notification = event.notification;
+  notification.close();
+  if (clients.openWindow) {
+    clients.openWindow('https://example.blog.com/2015/03/04/something-new.html');
```

Comment:
? what is this url

Reply:
Might be a copy/paste mistake. I'll take care of that ...